CMSgov/mint-app
# [NOREF] Split GraphQL Schema and GQL Resolvers into Multiple Files

# EASI-000

## Changes and Description

- Extracted types into individual schema files
- Extracted enums into a combined schema file



## How to test this change

This change necessarily affects all endpoints and must not break anything.
Smoke test endpoints and navigate through the website to assert that everything continues to function as expected.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
